fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerIndexFromIndexTest.java

```diff
@@ -382,6 +381,7 @@ public void testIndexFromIndexPersistentPreventBadContinuation() {
         try (OnlineIndexer indexBuilder = OnlineIndexer.newBuilder()
                 .setDatabase(fdb).setMetaData(metaData).setIndex(tgtIndex).setSubspace(subspace)
                 .setIndexStatePrecondition(OnlineIndexer.IndexStatePrecondition.BUILD_IF_DISABLED_CONTINUE_BUILD_IF_WRITE_ONLY_ERROR_IF_POLICY_CHANGED)
+                .setIndexingPolicy(OnlineIndexer.IndexingPolicy.newBuilder().build()) // this should be the default by-records
```

Comment:
Any reason this is explicitly setting the default? Just to have something for the comment to be next to? And should this use `OnlineIndexer.IndexingPolicy.DEFAULT` instead of `.newBuilder().build()`?

Reply:
Oh, I've just wished one of the tests to use this option to make sure there are no surprises in this path, another one is using the explicit `OnlineIndexer.IndexingPolicy.DEFAULT`.

Reply:
Updated the comments, hope it is less confusing now.